src/app/dynamic-forms/models/cinchy-form-section.model.ts

```diff
@@ -1,17 +1,30 @@
-import { FormField } from './cinchy-form-field.model';
+import { FormField } from "./cinchy-form-field.model";
+
+import { ILookupRecord } from "../../models/lookup-record.model";
+
 
 export class FormSection {
 
-  autoExpand: boolean;
   childFilter: string;
+
   childSort: string;
+
   columnsInRow: string;
+
   fields: Array<FormField> = [];
-  multiFields: Array<any>;
+
+  linkedColumnDetails: {
+    linkedElement: FormField,
+    linkLabel: string,
+    linkValue?: ILookupRecord
+  };
+
+  flattenedChildFormRecordValues: Array<any>;
```

Comment:
I know we spoke about this already, but just going to leave a comment here so I we can track on the PR itself. 
The name of this needs to change to avoid confusion with the actual flattening feature of child forms